src/app/services/technical-record/technical-record.service.ts

```diff
@@ -294,6 +294,29 @@ export class TechnicalRecordService {
       });
   }
 
+  generatePlate(vehicleRecord: VehicleTechRecordModel, techRecord: TechRecordModel, reason: string, user: { id?: string; name?: string }) {
+    const url = `${environment.VTM_API_URI}/vehicles/documents/plate`;
+
+    const body = {
+      vin: vehicleRecord.vin,
+      primaryVrm: techRecord.vehicleType !== 'trl' ? vehicleRecord.vrms.find(x => x.isPrimary)!.vrm : undefined,
+      systemNumber: vehicleRecord.systemNumber,
+      trailerId: techRecord.vehicleType === 'trl' ? vehicleRecord.trailerId : undefined,
+      msUserDetails: { msOid: user.id, msUser: user.name },
+      techRecord: vehicleRecord.techRecord,
+      reasonForCreation: reason,
+      vtmUsername: user.name
+    };
+
+    return this.http.post<VehicleTechRecordModel>(url, body, { responseType: 'json' });
+  }
+
+  generateLetter(techRecord: TechRecordModel, letterType: string) {
+    // TODO: Implement API call when ready
+    console.log('Piiing.');
```

Comment:
Is this console log being left in as a placeholder or something ? Or can it be removed ?

Reply:
Leaving it in as a placeholder, will be covering it this sprint iirc